<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>音频播放器</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="1" colspan="3">
     <widget class="CurvePlot" name="ploter" native="true"/>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="lbCurTime">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string> 00:00</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QSlider" name="slTime">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="4">
     <widget class="QListWidget" name="lwList">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget::item{height: 40px}

QListWidget {
	background-color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="lbTotalTime">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>00:00 </string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="CurvePlot" name="ploterAll" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="actionMinus"/>
   <addaction name="actionClear"/>
   <addaction name="actionList"/>
   <addaction name="separator"/>
   <addaction name="actionPlay"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
  </widget>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
   </property>
   <property name="text">
    <string>play</string>
   </property>
   <property name="toolTip">
    <string>播放</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images/pause.png</normaloff>:/images/pause.png</iconset>
   </property>
   <property name="text">
    <string>pause</string>
   </property>
   <property name="toolTip">
    <string>暂停</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
   <property name="toolTip">
    <string>停止</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images/plus.png</normaloff>:/images/plus.png</iconset>
   </property>
   <property name="text">
    <string>add</string>
   </property>
   <property name="toolTip">
    <string>增加</string>
   </property>
  </action>
  <action name="actionList">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images/list.png</normaloff>:/images/list.png</iconset>
   </property>
   <property name="text">
    <string>List</string>
   </property>
   <property name="toolTip">
    <string>列表</string>
   </property>
  </action>
  <action name="actionMinus">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images/minus.png</normaloff>:/images/minus.png</iconset>
   </property>
   <property name="text">
    <string>minus</string>
   </property>
   <property name="toolTip">
    <string>减去</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
   <property name="toolTip">
    <string>清除</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CurvePlot</class>
   <extends>QWidget</extends>
   <header>CurvePlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images/images.qrc"/>
 </resources>
 <connections/>
</ui>
